droidconKE/chai
# Laying screen foundation

---
name: Chai Demo foundations
about: Laying screen foundation in chaiDemo
label: 'triage me'
Fixes: #52 


---
## How can this PR be tested?
Checkout branch and run it.

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ x ] Ensure your branch is up to date with main branch.
- [ x ] Ensure your branch runs without breaking.
- [ x ] Ensure you solved any merge conflicts.
- [ x ] Smaller PRs for the ease of reviewing



# Screenshot

![image](https://github.com/user-attachments/assets/fd711dda-109c-4fe2-b01c-259ed97dde71)


![]()
